code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -326,8 +326,95 @@
 	volume = 50
 	amount_per_transfer_from_this = 10
 
+	var/cap_icon_state = "bottle_cap"
+	var/cap_on = TRUE
+	var/cap_lost = FALSE
+	var/mutable_appearance/cap_overlay
+	var/cap_x_offset = 0
+	var/cap_y_offset = -5 // small bottle is shorter
+
+
+/obj/item/reagent_containers/glass/beaker/waterbottle/Initialize()

```

Comment:
I hate this inheritance tree. I'm not sure what glass and beaker do but the fact that plastic bottles inherit from them worry me

Reply:
Out of scope for this PR

Reply:
Not really. If you're expanding the functionality of something it's generally also expected that you take in account that we don't want the shitty code to become even worse.